### DISCUSSION TITLE
W #1145

### DISCUSSION THREAD

**Discussion by Teacup1233** [share121/inter-knot#1145] [其他]
![和谐提示1@1x](https://github.com/user-attachments/assets/7f8486e6-81ef-4a42-99df-6c0570937ba0)

**Comment by kallen1314** [comment]
@治安局@治安局 这里貌似有人偷偷进空洞被侵蚀了！！快来救命啊！！希望来得及！

**Comment by Wchanging** [comment]
谁家以骸😡

**Comment by ZhuYuan-148** [comment]
依骸已被消灭，还请广大市民放心。
![Uploading Screenshot_20240810_180118.jpg…]()

**Reply by greenhoie** [reply]
图片要等一会再上传

**Comment by lllliufeng** [comment]
我去，空洞侵蚀到这了？

**Comment by zhangyj0523** [comment]
哥们空洞内怎么上的网啊

**Reply by fu-fu404** [reply]
我查过了，ip地址在芭莱大厦楼顶

**Comment by hanshui2077** [comment]
emm，是不是因为#1145太臭了导致的

**Comment by whsjjj28288** [comment]
应该是使用了邦布内置上网芯片！

**Comment by szlx2333** [comment]
笑死，楼主被空洞侵蚀和谐了

**Comment by lllliufeng** [comment]
我去，绳管高强度巡逻

**Comment by blqdswkl** [comment]
田所浩二实名上网（确信）

**Comment by 1451374815** [comment]
好臭，楼主有头绪吗
